site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -50,6 +50,9 @@ Configurable parameters:
 - region - the location of the cluster
 - node_count - count of game server nodes for the default node pool (default is "4")
 - feature_gates - a list of alpha and beta version features to enable. For example, "PlayerTracking=true&ContainerPortAllocation=true"
+- gameserver_minPort - the lower bound of the port range which gameservers will listen on (default is "7000")
```

Comment:
These should be version gated on the next agones release, right?

Reply:
https://agones.dev/site/docs/contribute/#within-a-page guide on feature gating within a page.